docs/ansible.utils.in_any_network_test.rst

```diff
@@ -0,0 +1,170 @@
+.. _ansible.utils.in_any_network_test:
+
+
+****************************
+ansible.utils.in_any_network
+****************************
+
+**Test if Test if an IP or network falls in any network**
+
+
+Version added: 2.0.1
```

Comment:
Since it is planned to go in next minor release the value of the version should be `2.1.0`

Reply:
Applicable at other places as well